### ISSUE TITLE
Create `development` branch and make it default

### ISSUE THREAD

**Issue by simonrnss** [AI-SDC/AI-SDC#6]
`main` branch should be kept for released code.

Create a `development` branch and make it the default.

**Comment by simonrnss** [comment]
Ah, there already is a `development` branch, it's just not default. I cannot see the options to modify the default so I guess I don't have the access -- could @jim-smith or @rpreen do it please?

**Comment by simonrnss** [comment]
Also, what do you think about requiring the two reviews to merge into development? I think it's pretty essential to ensure that things like docs + tests are sufficient?

**Comment by jim-smith** [comment]
made it default,  agree about two reviews in general, but perhaps not until  Andy is done setting up the docs which is a bit faffy.

CAn't see where to set 2 reviewer in settings ...

**Comment by simonrnss** [comment]
Good plan - I couldn't see it either - maybe it's a default on main?
Added another workflow pipeline to lint the code and also automatically run any tests on any push...

**Comment by simonrnss** [comment]
Going to close this -- we can revisit the number of reviewers later
